src/components/BookList.js

```diff
@@ -46,12 +50,20 @@ const BookList = () => {
         }).catch((err) => console.log('Error: ', err));
     }, []);
 
+    useEffect(() => {
```

Comment:
Is anyone aware of how I could test this? My goal for adding this is that the search box would be focused and ready to type when the page loads, but I'm not sure how this would be tested in RTL.

Reply:
There is a .toHaveFocus expectation.

Reply:
Great! I really need to look at all of the available expectations. I'll push an update to this PR this evening.